recipes/python-chromedriver-binary/bld.bat

```diff
@@ -0,0 +1,5 @@
+python download-chromedriver
+7z x chromedriver.zip -ochromedriver
```

Comment:
Also please add a `if errorlevel 1 exit 1` line here so it fails early

Reply:
This new conda-forge linter is pretty good

Reply:
😆